README.md

```diff
@@ -265,3 +265,42 @@ You will need the following items to run the sample:
    sudo apt update;
    sudo apt install --assume-yes dotnet-sdk-7.0;
    ```
+
+## Check out our other repos!
```

Comment:
Should this be top level (1 #) to match the Instructions and Troubleshooting sections?